staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/conflict.go

```diff
@@ -76,6 +76,9 @@ func printManager(manager string) string {
 		return fmt.Sprintf("%q", manager)
 	}
 	if encodedManager.Operation == metav1.ManagedFieldsOperationUpdate {
+		if encodedManager.Time == nil {
+			return fmt.Sprintf("%q using %v", encodedManager.Manager, encodedManager.APIVersion)
```

Comment:
Can we make this required instead?

Reply:
We don't fill it in for applies though right now. It depends on the operation if this will be there.

We will probably eventually add it for Apply too, but it wasn't trivial to implement, since we don't use it to build the manager info for apply, it would mean we can't just convert directly from smd format back into api format for the entries.